.github/workflows/docker-chatbot-rag-app.yml

```
@@ -49,21 +49,21 @@ jobs:
         with:
           context: example-apps/chatbot-rag-app
           # If this is a pull request, tag the image locally for testing.
-          tags: ${{ github.event_name == 'pull_request' && format('{0}:latest', env.IMAGE) || '' }}
+          tags: ${{ github.event_name == 'pull_request' && 'chatbot-rag-app:test' || '' }}
           # When pushing to main, push this image to the repository by digest
           # which is later used to assemble a multi-architecture image.
           outputs: type=image,name=${{ env.IMAGE }},push-by-digest=true,name-canonical=true,push=${{ github.event_name == 'push' && 'true' || 'false' }}
```

I think you need this

```suggestion
          outputs: type=${{ github.event_name == 'pull_request' && 'docker' || 'image' }},name=${{ env.IMAGE }},push-by-digest=true,name-canonical=true,push=${{ github.event_name == 'push' && 'true' || 'false' }}
```

https://github.com/anuraaga/elasticsearch-labs/actions/runs/13429737033/job/37519043395?pr=2